app/assets/javascripts/cqm/PopulationSet.js

```diff
@@ -30,6 +30,7 @@ const ObservationSchema = new mongoose.Schema({
   observation_function: StatementReferenceSchema,
   observation_parameter: StatementReferenceSchema,
   aggregation_type: String,
+  hqmf_id: String,
```

Comment:
I would suggest adding the hqmf_id to the stratification schema as well.

Reply:
added